bitnami/dataplatform-bp1/templates/exporter-deployment.yaml

```diff
@@ -139,13 +139,19 @@ spec:
           startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.dataplatform.exporter.customStartupProbe "context" $) | nindent 12 }}
           {{- end }}
           volumeMounts:
+            - name: exporter-config
+              mountPath: /data/bp.json 
+              subPath: bp.json
           {{- if .Values.dataplatform.exporter.extraVolumeMounts }}
           {{- include "common.tplvalues.render" (dict "value" .Values.dataplatform.exporter.extraVolumeMounts "context" $) | nindent 12 }}
           {{- end }}
         {{- if .Values.dataplatform.exporter.sidecars }}
         {{- include "common.tplvalues.render" ( dict "value" .Values.dataplatform.exporter.sidecars "context" $) | nindent 8 }}
         {{- end }}
       volumes:
+        - name: exporter-config
+          configMap:
+            name: {{ include "dataplatform.exporter-name" . }}-configuration
```

Comment:
```suggestion
            name: {{- include "dataplatform.exporter-name" . }}-configuration
```

Reply:
k8s throws an error if I change it to {{- include" dataplatform.exporter-name" . }}-configuration